doc/doxygen/bibliography.bib

```diff
@@ -511,7 +459,7 @@ @Article{reed92a
   doi                      = {10.1063/1.462145},
 }
 
-@Book{schlick2010,
+@Book{Schlick2010,
```

Comment:
This BibTeX file was created before the ICP existed. Several items here don't even exist in `icp.bib`. I had plans to re-organize that, but since we need to keep the python branch in sync with the walberla branch, it's easy to introduce merge conflicts (touching these keys means touching the C++ code). Considering the walberla branch itself needs to be kept in sync with large refactoring branches like lees-edwards, ekin and dynamic UBB, and that the python branch is also a base for the bond refactoring branch, I'm not sure it is worth the trouble. Maybe this could be done safely in the 4.2 release branch.

Reply:
I took care of a subset of keys that wouldn't cause merge conflicts (hopefully).